src/components/item.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import "../scss/item.scss";
+
+export default function Item() {
```

Comment:
Item is singular, Items is plural
This component is meant to be Item, which means that it will render only one item.